go.mod

```diff
@@ -3,9 +3,13 @@ module github.com/jaegertracing/jaeger-opentelemetry-collector
 go 1.13
 
 require (
+	github.com/jaegertracing/jaeger v1.14.0
```

Comment:
Any reason this is `v1.14.0`?
If the aim is to replace the collector in Jaeger, then at some point (possibly not now) the dependency on jaeger could be removed, and the actual ES client code moved into this exporter? Do we want to limit the packages pulled in?

Reply:
no reason this is what go resolved by default.